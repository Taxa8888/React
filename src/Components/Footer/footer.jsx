import React from 'react';
import './footer.style.scss';
import { MAIN_PROJECT_TITLE } from '../app/app.constants';

const Footer = ({footerBgImg}) => (
    <header className="footer" style={{backgroundImage: `url(${footerBgImg})`}}>
        <div className="footerContainer">
            <p className="footerTitle">{MAIN_PROJECT_TITLE}</p>
        </div>
    </header>
);

export default Footer;
