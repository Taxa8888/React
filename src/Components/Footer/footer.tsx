import React, { ReactElement } from 'react';
import { MAIN_PROJECT_TITLE } from '../app1/app.constants';
import { FOOTER_BG_IMAGE } from './footer.constants';
import './footer.style.scss';

export const Footer = (): ReactElement => (
    <header className="footer" style={FOOTER_BG_IMAGE}>
        <div className="footerContainer">
            <p className="footerTitle">{MAIN_PROJECT_TITLE}</p>
        </div>
    </header>
);
