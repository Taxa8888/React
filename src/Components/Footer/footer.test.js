import { shallow } from 'enzyme';
import React from 'react';
import { Footer } from './footer';

/*global describe, it, expect*/

describe('Footer component', () => {
    it('should render Footer component', () => {
        const component = shallow(<Footer />);
        expect(component).toMatchSnapshot();
    });
});
