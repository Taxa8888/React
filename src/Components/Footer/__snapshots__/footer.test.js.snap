// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should render Footer component 1`] = `
<header
  className="footer"
  style={
    Object {
      "backgroundImage": "url(test-file-stub)",
    }
  }
>
  <div
    className="footerContainer"
  >
    <p
      className="footerTitle"
    >
      netflixroulette
    </p>
  </div>
</header>
`;
